cli/src/org/partiql/cli/functions/ReadFile.kt

```diff
@@ -76,9 +84,21 @@ internal class ReadFile(valueFactory: ExprValueFactory) : BaseFunction(valueFact
         "customized" to fileReadHandler(CSVFormat.DEFAULT)
     )
 
-    override fun call(env: Environment, args: List<ExprValue>): ExprValue {
-        val options = optionsStruct(1, args)
-        val fileName = args[0].stringValue()
+    override fun callWithRequired(env: Environment, required: List<ExprValue>): ExprValue {
+        val fileName = required[0].stringValue()
+        val fileType = "ion"
+        val handler = readHandlers[fileType] ?: throw IllegalArgumentException("Unknown file type: $fileType")
+        val seq = Sequence {
+            // TODO we should take care to clean this up properly
```

Comment:
Not a blocker for this PR but as a fast follow: can we clarify further with linking to an issue preferably?